src/inverseRobot.js

```diff
@@ -7,7 +7,17 @@
 */
 
 function inverseRobot(robot) {
-  // write code here
+  const invertedRobot = {};
+
+  for (const key in robot) {
+    if (robot[key] in invertedRobot) {
+      return null;
+    }
+
+    invertedRobot[robot[key]] = key;
+  }
```

Comment:
Consider using 'Object.hasOwnProperty()' to check if a property exists on an object. While using 'in' is not incorrect, 'hasOwnProperty()' does not check down the object's prototype chain, which can be safer for property existence checks.